kata.py

```diff
@@ -12,18 +19,34 @@ def Empty(string):
                 else:
                     answer = 0
                     for num in string[4:].split('\n'):
+                        Negative(num)
                         answer += int(num)
-                    return answer
+                    if len(negatives_list) == 0:
+                        return answer
+                    else:
+                        raise Exception(f'Negatives not allowed: {negatives_list}')
             elif Empty(string.split('\n')[1]):
                 return 0
             else:
                 answer = 0
                 delimiter = string.split('\n')[0].strip('//')
                 for num in string.split('\n')[1].split(delimiter):
+                    Negative(num)
                     answer += int(num)
-                return answer
+                if len(negatives_list) == 0:
+                    return answer
+                else:
+                    raise Exception(f'Negatives not allowed: {negatives_list}')
```

Comment:
I probably need to refresh my Python knowledge a little before writing this, but... `Exception` in most cases is probably a bit too broad - I'd normally expect to see a custom exception inherit from this as its base class. If you get more specific with your exceptions you can make them more specific. This might potentially accidentally trigger false positives when you get to the test part as I think _everything_ inherits from `Exception`.